scripts/script.js

```diff
@@ -0,0 +1,178 @@
+function displayAllSpeakers() {
+  speakersSection.classList.toggle('active');
+  speakersUl.classList.toggle('active');
+  let list = '';
+  featuredSpeakersArr.forEach((speaker) => {
+    list = `${list}<li class="featured-speakers-list-item">
+            <div class="featured-speakers-list-item-img">
+                <img src= "${speaker.img}" alt="Speaker 1 image">
+            </div>
+            <div class="featured-speaker-info">
+                <h3 class="featured-speaker-name"> ${speaker.name}</h3>
+                <p class="featured-speaker-description">
+                    ${speaker.profession}
+                </p>
+                <div class="speaker-separator"> </div>
+                <p class="speaker-studies">
+                    ${speaker.work}
+                </p>
+            </div>
+            </li>`;
+  });
+  speakersUl.innerHTML = list;
+  speakersSection.appendChild(speakersUl);
+//   showMoreSpeakersBtnState = 1;
+}
+
+// function displayLessSpeakers() {
+//   const list = '';
+//   if (showMoreSpeakersBtnState === 1) {
+//     console.log('Enter');
+//     displayFeaturedSpeakers();
+//     showMoreSpeakersBtnState = 0;
+//     // featuredSpeakersArr.forEach((speaker, index) => {
+//     //     if (index < 2) {
+//     //         list = `${list}<li class="featured-speakers-list-item">
+//     //     <div class="featured-speakers-list-item-img">
+//     //         <img src= "${speaker.img}" alt="Speaker 1 image">
+//     //     </div>
+//     //     <div class="featured-speaker-info">
+//     //         <h3 class="featured-speaker-name"> ${speaker.name}</h3>
+//     //         <p class="featured-speaker-description">
+//     //             ${speaker.profession}
+//     //         </p>
+//     //         <div class="speaker-separator"> </div>
+//     //         <p class="speaker-studies">
+//     //             ${speaker.work}
+//     //         </p>
+//     //     </div>
+//     //     </li>`;
+//     //     } else {
+//     //         list = `${list}<li class="featured-speakers-list-item
+// featured-speakers-list-item-pc">
+//     //     <div class="featured-speakers-list-item-img">
+//     //         <img src= "${speaker.img}" alt="Speaker 1 image">
+//     //     </div>
+//     //     <div class="featured-speaker-info">
+//     //         <h3 class="featured-speaker-name"> ${speaker.name}</h3>
+//     //         <p class="featured-speaker-description">
+//     //             ${speaker.profession}
+//     //         </p>
+//     //         <div class="speaker-separator"> </div>
+//     //         <p class="speaker-studies">
+//     //             ${speaker.work}
+//     //         </p>
+//     //     </div>
+//     //     </li>`;
+//     //     }
+//     // });
+//     // speakersSection.classList.toggle('active');
+//     // speakersUl.classList.toggle('active');
+//     // showMoreSpeakersBtnState = 0;
+//     // speakersUl.innerHTML = list;
+//     // speakersSection.appendChild(speakersUl);
+//   }
+// }
```

Comment:
- [ ] I kindly suggest that your remove commented out code from your code-base to make your work neater.